userspace/libsinsp/threadinfo.cpp

```diff
@@ -2178,7 +2178,7 @@ threadinfo_map_t::ptr_t sinsp_thread_manager::find_thread(int64_t tid, bool look
 
 void sinsp_thread_manager::set_max_thread_table_size(uint32_t value)
 {
-    m_max_thread_table_size = std::min(value, m_thread_table_absolute_max_size);
+    m_max_thread_table_size = value;
```

Comment:
Am not sure whether we want to retain some sort of `max` value here.

Reply:
How would we choose the absolute new max value in a way that wouldn't limit flexibility?

Reply:
Yes fair point! I am not sure , like 10x default value? But of course that would still be an hard limit...

Reply:
We have UINT32 max value as super hard limit ...

Reply:
Let's go as is then! I was just thinking loud :)